cf/generate-staging.sh

```diff
@@ -13,3 +13,4 @@ spiff merge \
 sed -i -- 's/10.10/10.9/g' manifest-staging.yml
 # Reverts this IP back for the cg-metrics
 sed -i -- 's/10.9.101.63/10.10.101.63/g' manifest-staging.yml
+sed -i -- 's/10.9.1.7\n/10.10.1.7\n/g' manifest-staging.yml
```

Comment:
Also, I'll have a fix for this soon enough that should work: https://github.com/18F/cg-manifests/compare/master...LinuxBozo:generate-pathmod#diff-2e8309a523569b3385f5b831e92a56c2R30